odoo_module_migrate/migration_scripts/migrate_150_160.py

```diff
@@ -2,6 +2,16 @@
 
 from odoo_module_migrate.base_migration_script import BaseMigrationScript
 
+_REMOVED_FIELDS = [
+    ('product.product', 'price',
+        'Commit https://github.com/odoo/odoo/commit/9e99a9df464d97a74ca320d')
```

Comment:
Hola Jose!

Please consider multi-files style for the new config. See #42 
Also, update DEVELOP.rst file

Reply:
Hi @yelizariev !
Great to meet you againt and Odoo XP. Thanks for the review and recommendations! Nice shot.
We'll work on the changes.

Thanks!

Reply:
Hey @yelizariev @jjscarafia 

I added a commit moving the new helpers to separate folders and updating the DEVELOP.rst, could you take a look?